### DISCUSSION TITLE
GBA Doesn't run the first time ONLY ON WINDOWS

### DISCUSSION THREAD

**Discussion by KirstenAli** [EmulatorJS/EmulatorJS#407] [Q&A]
The first time you run a gba game it does not work and the page stalls. It works the second try. It also works perfectly on my mac and Pi Chromium. I'm only experiencing this problem on windows (Chrome, edge, Firefox).

**Comment by ethanaobrien** [comment]
I've noticed this, it seems 100% random (except I've only seen it on windows, hadn't realized that until you brought it up) and there's no apparent fix other than to reload the page.

Even using the debugger, it doesn't give me any helpful info as it just seems to freeze

This might be a bug within the os itself, when I get a chance I'll open an issue on the chromium bug page and see if they might know why this happens

**Reply by KirstenAli** [reply]
I've noticed Vimm's Lair seems to have got this working. They seem to use EmulatorJS and it works perfectly each time regardless of OS. I've tried swapping their loader.js, emulator.min.js, etc. but no luck.
